package.json

```diff
@@ -8,8 +8,35 @@
   "bugs": {
     "url": "https://github.com/muxinc/media-chrome/issues"
   },
-  "type": "module",
   "main": "dist/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/cjs/index.js",
+      "default": "./dist/index.js"
+    },
+    "./react": {
+      "import": "./dist/react/index.js",
+      "require": "./dist/react/cjs/index.js",
+      "default": "./dist/react/index.js"
+    },
+    "./dist/react/*": {
+      "import": "./dist/react/*",
+      "require": "./dist/react/cjs/*",
+      "default": "./dist/react/*"
+    },
+    "./dist/*": {
+      "import": "./dist/*",
+      "require": "./dist/cjs/*",
+      "default": "./dist/*"
+    },
+    "./*": {
```

Comment:
allow for import as `media-chrome/media-play-button.js`

Reply:
yes, this one I’m a bit worried if we have the bundler conveniences and that code somehow got used in the browser w/ imports and it would not work.